spec/requests/seance_spec.rb

```diff
@@ -48,11 +48,13 @@
 
   describe "POST /seances" do
     subject(:request) { post('/seances', params:) }
-    
+
     let(:movie) { create :movie }
     let(:hall) { create :hall }
 
-    let(:params) { { seance: attributes_for(:seance, start_time: DateTime.current, price:, movie_id: movie.id, hall_id: hall.id ) } }
+    let(:params) do
+      { seance: attributes_for(:seance, start_time: DateTime.current, price:, movie_id: movie.id, hall_id: hall.id) }
```

Comment:
O - I didn't know about `attributes_for` of FactoryBot, you learn something new everyday 😄